Autocoders/Python/schema/testSchemas.py

```diff
@@ -249,7 +249,7 @@ def setup():
         AssertionError,
     )
     command_test.add_test(
-        "Kind not sync nor async",
+        "Kind neither sync nor async",
```

Comment:
There's a heuristic that flags `not/nor`, I've applied this change, but left the heuristic off because I didn't have the energy to address the other entries it flagged and I'm less confident in it for the longer cases it found.